src/data/builder-tools.js

```diff
@@ -180,6 +180,62 @@ export const Tags = {
 // Add your builder tool to (THE END OF) this list.
 // Please don't add the "favorite"-tag yourself.
 const Showcases = [
+  {
+    title: "cardano-cli",
+    description: "The companion command-line to interact with a Cardano node, manipulate addresses or create transactions.",
+    preview: require("./builder-tools/cardano-cli.png"),
+    website: "https://github.com/input-output-hk/cardano-node/tree/master/cardano-cli#cardano-cli",
+    getstarted: null,
+    tags: ["favorite", "cli"]
+  },
+  {
+    title: "bech32",
+    description: "Convert to and from bech32 strings from the command-line. A simple and easy-to-use unix utility.",
```

Comment:
Not sure if this deserved to be a seperate entry, as it's quite basic utility

Reply:
I tend to agree...  I only put it there because it was listed as one of the 'Cardano components'  😬

It's basic utility, probably doesn't deserve a spot in the builder tools but simply, mentioned somewhere in one of the guide as some useful tool to have on one's Swiss knife.